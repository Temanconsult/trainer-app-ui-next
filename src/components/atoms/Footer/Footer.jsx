import React from 'react'
import { Card, Col, Row, ListGroup } from 'react-bootstrap'

const Footer = () => {
  return (
    <Card className="mt-2 footer" text="white" bg="dark">
      <Card.Body>
        <Row>
          <Col sm={12} md={6} lg={5} className="footer-section">
            <Card.Title>Best Tutor</Card.Title>
            <Card.Text>
              Best tutor provides the platform for instructors and students to
              connect. Instructors will have the opportunity to teach the topics
              of their expertise while students get to learn their desired
              courses from the best minds around. Sign up today and kick off
              that opportunity to start a new carrer or learn more about what
              you already know.
            </Card.Text>
          </Col>
          <Col sm={12} md={3} lg={4} className="footer-section">
            <Card.Title>Contact Us</Card.Title>
            <Card.Text>
              <ListGroup variant="flush">
                <ListGroup.Item>Phone: +1-612-402-1495</ListGroup.Item>
                <ListGroup.Item>Email: info@besttutor.com</ListGroup.Item>
              </ListGroup>
            </Card.Text>
          </Col>
          <Col sm={12} md={3} lg={3} className="footer-section">
            <Card.Title>Our Courses</Card.Title>
            <Card.Text>
              <ListGroup variant="flush">
                <ListGroup.Item>Scrum Master</ListGroup.Item>
                <ListGroup.Item>Project Management</ListGroup.Item>
                <ListGroup.Item>JavaScript</ListGroup.Item>
                <ListGroup.Item>DevOps</ListGroup.Item>
              </ListGroup>
            </Card.Text>
          </Col>
        </Row>
      </Card.Body>
    </Card>
  )
}

export default Footer
