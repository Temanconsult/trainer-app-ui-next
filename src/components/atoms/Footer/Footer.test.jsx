import React from 'react'
import { render } from '@testing-library/react'
import Footer from '.'

describe('Footer', () => {
  it('creates footer', () => {
    const { asFragment } = render(<Footer />)
    expect(asFragment()).toMatchSnapshot()
  })
})
