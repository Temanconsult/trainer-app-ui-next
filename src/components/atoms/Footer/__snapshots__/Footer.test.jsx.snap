// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer creates footer 1`] = `
<DocumentFragment>
  <div
    class="mt-2 footer card bg-dark text-white"
  >
    <div
      class="card-body"
    >
      <div
        class="row"
      >
        <div
          class="footer-section col-lg-5 col-md-6 col-sm-12"
        >
          <div
            class="card-title h5"
          >
            Best Tutor
          </div>
          <p
            class="card-text"
          >
            Best tutor provides the platform for instructors and students to connect. Instructors will have the opportunity to teach the topics of their expertise while students get to learn their desired courses from the best minds around. Sign up today and kick off that opportunity to start a new carrer or learn more about what you already know.
          </p>
        </div>
        <div
          class="footer-section col-lg-4 col-md-3 col-sm-12"
        >
          <div
            class="card-title h5"
          >
            Contact Us
          </div>
          <div
            class="list-group list-group-flush"
          >
            <div
              class="list-group-item"
            >
              Phone: +1-612-402-1495
            </div>
            <div
              class="list-group-item"
            >
              Email: info@besttutor.com
            </div>
          </div>
        </div>
        <div
          class="footer-section col-lg-3 col-md-3 col-sm-12"
        >
          <div
            class="card-title h5"
          >
            Our Courses
          </div>
          <div
            class="list-group list-group-flush"
          >
            <div
              class="list-group-item"
            >
              Scrum Master
            </div>
            <div
              class="list-group-item"
            >
              Project Management
            </div>
            <div
              class="list-group-item"
            >
              JavaScript
            </div>
            <div
              class="list-group-item"
            >
              DevOps
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
